coreneuron/nrniv/memory.h

```diff
@@ -57,23 +103,22 @@ inline bool is_aligned(void* pointer, size_t alignment) {
 
 /** Allocate the aligned memory.
  */
-inline void* emalloc_align(size_t size, size_t alignment) {
+inline void* emalloc_align(size_t size, size_t alignment = NRN_SOA_BYTE_ALIGN) {
     void* memptr;
-    nrn_assert(posix_memalign(&memptr, alignment, size) == 0);
+    alloc_memory(memptr, size, alignment);
     nrn_assert(is_aligned(memptr, alignment));
     return memptr;
 }
 
 /** Allocate the aligned memory and set it to 1.
```

Comment:
> // Avoid native division by zero (cyme...)

In the old days, these allocated vectors were used with library (`cyme`). If "some" element of the vector is 0, it will be an error. And hence it was set to 1 to avoid this error.

This is no loner required. So I would keep this semantically correct with calloc i.e. 0.